src/heist_path/HeistPathB.py

```
@@ -307,18 +306,195 @@ def lg1_wp4_buster(self):
+        self.sleep(0.28)
+        self.send_key_down("w")
+        self.sleep(2.56)
+        self.send_key_down("a")
+        self.sleep(0.30)
+        self.send_key_up("a")
+        self.sleep(1.57)
+        self.exit_state[2] = self.try_open_exit(direction="w")
+        self.sleep(0.40)
+
+    # 尝试兼容1600*900分辨率
+    def check_current_floor_str(self, str):
```

Ruff (0.15.13)

[error] Function argument `str` is shadowing a Python builtin

(A002)